docs/user/reference/general/register-custom-pings.md

```diff
@@ -77,8 +77,28 @@ pings = load_pings(resource_filename(__name__, "pings.yaml"))
 
 </div>
 
-<div data-lang="Rust" class="tab"></div>
+<div data-lang="Rust" class="tab">
+
+In Rust custom pings need to be registered individually.
+This should be done before calling `glean::initialize`.
+
+```Rust
+use your_glean_metrics::pings;
+
+glean::register_ping_type(&pings::custom_ping);
```

Comment:
nit: Maybe register two different pings in this example so it is clear pings need to be registered individually?